zephyr/test/userspace.c

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
likewise re names - my feeling is we will have more userspace tests so best to say userspace-locking.c